tests/suites/1_stateful/03_presign/03_0000_presign_download.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. "$CURDIR"/../../../shell_env.sh
+
+echo "drop stage if exists presign_stage" | $MYSQL_CLIENT_CONNECT
+
+aws --endpoint-url http://127.0.0.1:9900/ s3 cp s3://testbucket/admin/data/ontime_200.csv s3://testbucket/admin/stage/presign_stage/ontime_200.csv >/dev/null 2>&1
+
+echo "CREATE STAGE presign_stage;" | $MYSQL_CLIENT_CONNECT
```

Comment:
What will happen if we create an external stage, and get it pregisn?

Reply:
presign will generate a URL with the external stage's credential.